### DISCUSSION TITLE
Not able to see Google pay as a payment method in android

### DISCUSSION THREAD

**Discussion by shhbzkhn** [flutter-stripe/flutter_stripe#1531] [General]
**Describe the bug**
Not able to see Google pay as a payment method in android

**To Reproduce**
when you open stripe initialisePaymentSheet there is no google pay available as a payemnt option

**Expected behavior**
It should google pay as a payment option

**Smartphone / tablet**
 - Device: [e.g. google pixel 7,  google pixel 3A, google pixel 4, samsung a52]
 - OS: [Android 13]
 - Package version: [flutter_stripe: ^9.6.0]
 - Flutter version [3.16.2]

![Screenshot-1204-110815](https://github.com/flutter-stripe/flutter_stripe/assets/104189882/bea9618d-1eeb-4712-8f0e-b5462e2f26f0)

Here is the code that i have used.

    Future<Future<PaymentSheetPaymentOption?>> initialisePaymentSheet(
        StripePaymentIntentResponseModel stripeIntentData,
        double? orderTotal) async {
      final billingDetails = BillingDetails(
        name: SharedPref.getString(SharedPref.userNameKey.split(' ').first),
        email: SharedPref.getString(SharedPref.emailKey),
        phone: getUSFormatted(SharedPref.getString(SharedPref.phoneNumberKey)),
        address: Address(
          city: SharedPref.getString(SharedPref.city),
          country: SharedPref.getString(SharedPref.city),
          line1: SharedPref.getString(SharedPref.street),
          line2: '',
          state: SharedPref.getString(SharedPref.state),
          postalCode: SharedPref.getString(SharedPref.postal_code),
        ),
      );
      return Stripe.instance.initPaymentSheet(
        paymentSheetParameters: SetupPaymentSheetParameters(
          // Main params
          paymentIntentClientSecret:
              stripeIntentData.tokenData?.clientSecret.toString(),
          merchantDisplayName: ' SeatEats Marketplace',
          // Customer params
          // customerId: data.tokenData?.id.toString(),
          customerEphemeralKeySecret:
              stripeIntentData.tokenData?.emphData?.id.toString(),
          style: ThemeMode.light,
          billingDetails: billingDetails,
          //applePay
          applePay: const PaymentSheetApplePay(merchantCountryCode: 'US'),

          googlePay: PaymentSheetGooglePay(
              merchantCountryCode: 'US',
              currencyCode: "USD",
              label: "Order Total",
              amount: orderTotal.toString(),
              testEnv: true),
        ),
      );
    }

**Comment by naseefvaliyakath** [comment]
In some countries stripe not support google pay and apple pay
https://support.stripe.com/questions/supported-payment-methods-currencies-and-businesses-for-stripe-accounts-in-india
Pleas check is your countrie is supported or not
You can separately integrate google pay with 
Pay dependency from pub.dev
Thanks
